src/App.js

```diff
@@ -1 +1,19 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Signup from './components/signup';
+import Signin from './components/signin';
 
+
+function App() {
+  return (
+    <Router>
+        <Switch>
+            <Route exact path="/"
```

Comment:
I think it's better the default should be the signin view then we can link the user to the signup page if they don't yet have an account

Reply:
Done

Reply:
Done locally? I've not seen the changes in the remote branch